components/youtube/youtube.app.js

```diff
@@ -0,0 +1,28 @@
+module.exports = {
```

Comment:
Two things on this file:

* You'll want to require `axios` in this file
* Can you also adjust the indentation of this file?